src/style.css

```diff
@@ -0,0 +1,108 @@
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+body {
+  margin: 0 8px;
+  font-weight: 300px;
+}
+
+.block {
+  margin-top: 20px;
+}
+
+.search {
+  font-family: Avenir, sans-serif;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  background-color: #fff;
+  color: #3d4e61;
+  box-sizing: border-box;
+}
+
+.searchBig {
+  height: 70px;
+  background-image: url("images/Search.svg");
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+  background-repeat: no-repeat;
+  background-position: 26px center;
+  background-size: 19px;
+}
+
+.searchSmall {
+  background-image: url("images/Search.svg");
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+  font-style: normal;
+  font-weight: normal;
+  line-height: 19px;
+  background-repeat: no-repeat;
+  background-position: 13px center;
+  background-size: 11px;
+}
+
+.search::-webkit-input-placeholder {
+  color: #555;
+}
+
+.search:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.search:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  border-radius: 3px;
+}
+
+.searchBig:hover {
+  height: 70px;
+  background-image: url("images/Search.svg");
+  padding-left: 62px;
+  background-repeat: no-repeat;
+  background-position: 26px center;
+  background-size: 19px;
+}
+
+.searchSmall:hover {
+  background-image: url("images/Search.svg");
+  height: 42px;
+  padding-left: 33px;
+  background-repeat: no-repeat;
+  background-position: 13px center;
+  background-size: 11px;
+}
+
+.searchBig:focus {
+  height: 70px;
+  background-image: url("images/Search.svg");
+  padding-left: 62px;
+  background-repeat: no-repeat;
+  background-position: 26px center;
+  background-size: 19px;
+}
```

Comment:
check with design, you need to add `background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);` on focus and other styles to make your default input look like this one when focused
<img width="638" alt="Screenshot 2021-12-29 at 19 04 50" src="https://user-images.githubusercontent.com/57598825/147686459-143f3591-91a4-4c70-9b5c-01ee39ee6dc0.png">